### ISSUE TITLE
[AdaptiveCards- Designer]: Focus Order is not in sequential order for the header controls on the 'Designer Page'.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8176]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: [https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0) 
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open above URL with valid credential.
2. Navigate to 'Designer' link and press enter key to activate it.
3. Navigate to 'Redo' button and press tab key and observe the issue.

**Actual Result:**
While navigating to the 'Designer' page focus order is not in sequential order on the header control as after the 'Redo' button on pressing the Tab key focus is going on the 'Copy card Payload' button.

**Expected Result:**
Focus order should be in logical order and after the 'Redo' button on pressing Tab key focus should be land on the Target Version combo box.

**User Experience:**
It will impact the keyboard user if the focus order is not in logical order as user have to navigate more to interact with desired elements.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B80A4F009-8781-4423-B3CD-2448EAF5BF62%7D&file=MAS%202.4.3%20%E2%80%93%20Focus%20Order.docx&action=default&mobileredirect=true

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/95913748/208826800-64d9e891-fb15-429f-a06d-63b810f8a72f.mp4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @anna-dingler,

As this issue is still repro, so could you please let us know is there any future plan to fix this issue?

**Comment by anna-dingler** [comment]
Closing this issue since it is a duplicate of https://github.com/microsoft/AdaptiveCards/issues/7282

**Comment by vagpt** [comment]
Adding A11yTTvalidated tag as per the comment above.
